inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -87,10 +87,10 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     common_fusions->add_matcher<ngraph::pass::SoftPlusFusion>();
     common_fusions->add_matcher<ngraph::pass::SoftPlusToMishFusion>();
     common_fusions->add_matcher<ngraph::pass::SwishFusion>();
+    common_fusions->add_matcher<ngraph::pass::ClampFusion>();
```

Comment:
This doesn't make sense because all matchers registered inside common_fusions apply simultaneously. So in case if ClampFusion  successfully applies then HSwishFusionWithClamp and HSigmoidFusionWithClamp matcher passes will successfully match and replace sub-graph with newly created Clamp operation.

Reply:
And BTW if you make such changes then please provide some unit-tests that proves that something wasn't working.